app/src/App.tsx

```diff
@@ -65,17 +64,14 @@ export const App = () => {
 
   React.useEffect(() => {
     const fetch = async () => {
-      // if (isAuthenticated) {
-      //   let fetch = new Fetcher<Host>('checkEmail')
-      //   let data: any | undefined = await fetch.getByEmail(user) //change any to account interface
-      //   if (data?.email === user?.email) { //change to status if status 200 then its gooooooood!
-      //     setHasAccount(data !== null)
-      //   }
-      //   history.push('profileselection')
-      // }
-      if(isAuthenticated){
+      if (isAuthenticated) {
         let fetching = new ApiWrapper()
-        let data: Accounts | undefined = await fetching.getUserAccount(user)
+        let data: any | undefined = await fetching.getUserAccount(user)
```

Comment:
In a future iteration, we will discuss naming conventions that will make the work you did more easily discoverable by team